features/support/steps.js

```diff
@@ -0,0 +1,112 @@
+const os = require('os');
+const path = require('path');
+const util = require('util');
+const childProcess = require('child_process');
+const { expect } = require('chai');
+const fs = require('fs-extra');
+const net = require('net');
+const which = require('which');
+const kill = require('tree-kill');
+const { Given, When, Then, Before, After } = require('cucumber');
+
+
+Before(function () {
```

Comment:
In our ES6 style guide, this and all the closures below are not permitted as they miss function names.